perseo-main/src/test/java/com/telefonica/iot/perseo/test/Help.java

```diff
@@ -39,46 +42,74 @@
 
     public static final int PORT = 8129;
 
-    public static final String[] ExampleRules = new String[]{
-        "select id, price? as Price from iotEvent.win:length(100) group by id",
-        "@Audit select *,\"blood_1_action\" as iotcepaction,"
-        + "ev.BloodPressure? as Pression, ev.id? as Meter from pattern "
-        + "[every ev=iotEvent(cast(cast(BloodPressure?,String),float)>1.5"
-        + " and type=\"BloodMeter\")]"};
-    public static final String[] ExampleNotices = new String[]{
-        "{\n"
-        + "\"BloodPressure\": 2,\n"
-        + "\"id\":\"guay!\",\n"
-        + "\"otro\":\"mas\",\n"
-        + "\"numero\":4,\n"
-        + "\"sub\": {\n"
-        + "	\"subnumero\":18,\n"
-        + "	\"subcadena\":\"SUB2\",\n"
-        + "	\"subflotante\": 12.3,\n"
-        + "	\"sub2\": { \"valor\": 3}\n"
-        + "	}\n"
-        + "}"};
-
+  
+    public static String[] ExampleRules() {
+    	return new String[]{
+    			 "select id, price? as Price from iotEvent.win:length(100) group by id",
+    	            "@Audit select *,\"blood_1_action\" as iotcepaction,"
+    	            + "ev.BloodPressure? as Pression, ev.id? as Meter from pattern "
+    	            + "[every ev=iotEvent(cast(cast(BloodPressure?,String),float)>1.5"
+    	            + " and type=\"BloodMeter\")]"};
+    	
+    }
+    public static String[] ExampleNotices() {
+    	return new String[]{
+                "{\n"
+                        + "\"BloodPressure\": 2,\n"
+                        + "\"id\":\"guay!\",\n"
+                        + "\"otro\":\"mas\",\n"
+                        + "\"numero\":4,\n"
+                        + "\"sub\": {\n"
+                        + "	\"subnumero\":18,\n"
+                        + "	\"subcadena\":\"SUB2\",\n"
+                        + "	\"subflotante\": 12.3,\n"
+                        + "	\"sub2\": { \"valor\": 3}\n"
+                        + "	}\n"
+                        + "}"};
```

Comment:
Ident seems a bit crazy in this part...

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823